src/YUILoader.cc

```diff
@@ -156,6 +157,34 @@ void YUILoader::loadUI( bool withThreads )
     }
 }
 
+void YUILoader::loadTestFrameworkPlugin( )
+{
+        if( !getenv("YUI_HTTP_PORT") ) return;
+        yuiMilestone () << "Requested to start http server to control UI." << std::endl;
+
+        if( pluginExists( YUIPlugin_Test ) ) {
+                YUIPlugin uiPlugin( YUIPlugin_Test );
+                if ( uiPlugin.success() )
+                {
+                        yuiMilestone () << "Loading http server to control UI." << std::endl;
+                        typedef void (*getServerFunction_t)();
```

Comment:
I'm not a C++ expert so I'm not sure how much is this local `typedef` a good solution...

Reply:
Right, will move it to the header file.